src/sass/components/_card.scss

```diff
@@ -161,18 +161,31 @@
     }
 
     &__candidate-read-more-link {
-      left: 0;
       margin-left: 10px;
       top: 0;
-      font-weight: lighter;
+      font-family: $body-font-stack;
+      font-size: 14px;
+      font-weight: 100 !important;
+      color: $link-color;
```

Comment:
@ixley it seems fine to me to wrap the links we want to show an underline on hover to use the a tag.